packages/application-shell/src/components/setup-flop-flip-provider/setup-flop-flip-provider.tsx

```diff
@@ -25,6 +59,7 @@ type TSetupFlopFlipProviderProps = {
   ldClientSideId?: string;
   children: ReactNode;
   shouldDeferAdapterConfiguration?: boolean;
+  shouldWaitForRemoteFlags?: boolean;
```

Comment:
This prop is included so it will be easier to disable the render delaying.
Instead of removing this code, release a new `app-kit` version and update consumer applications, we will just need to update consumer apps with this prop.
Then we can work in rolling back these changes without rushing.